scripts/cmake-configurator/parser/test_cmakeparser.py

```diff
@@ -0,0 +1,45 @@
+from cmake_parser import cmake_parser
+from unittest import TestCase
+
+file_lines_generic = """
+# KEY:TYPE=VALUE
+# KEY is the name of a variable in the cache.
+# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
+# VALUE is the current value for the KEY.
+
+########################
+# EXTERNAL cache entries
+########################
+
+//Build unit tests for the target arch
+ARCH_GTESTS:BOOL=OFF
+
+//Build the EFI bootloader
+BOOTLOADER_EFI:BOOL=ON
+
+//Build the legacy bootloader
+BOOTLOADER_LEGACY:BOOL=ON
```

Comment:
It would be cooler to be exhaustive at least for the booleans: just create a new variable for each possible bool value. There will be just 6 of them. You can call them BOOLVAR1, BOOLVAR2 etc. Then in the tests, check that all the logical true values map to true and all the logical false values map to false.